cmd/adapter-rest/startcmd/start.go

```diff
@@ -868,8 +868,8 @@ func addIssuerHandlers(parameters *adapterRestParameters, framework *aries.Aries
 	if err != nil {
 		return fmt.Errorf("aries-framework - failed to get aries context : %w", err)
 	}
-	// TODO #572 Pass the output descriptors to issuer
-	_, err = readCMOutputDescriptorFile(parameters.cmOutputDescriptorsFilePath)
+
+	CMOutputDescriptor, err := readCMOutputDescriptorFile(parameters.cmOutputDescriptorsFilePath)
```

Comment:
Shouldn't this local var name start with lower case ?

Reply:
done